cpp/src/arrow/csv/parser-benchmark.cc

```diff
@@ -28,44 +28,36 @@
 namespace arrow {
 namespace csv {
 
-static std::string BuildQuotedData(int32_t num_rows = 10000) {
-  std::string one_row = "abc,\"d,f\",12.34,\n";
-  std::stringstream ss;
-  for (int32_t i = 0; i < num_rows; ++i) {
-    ss << one_row;
-  }
-  return ss.str();
-}
+// Can't have static str.
```

Comment:
Hmm... why? You could use `const std::string`.

Reply:
Due to linter, I'll add a comment.